applications/openshift/kubelet/kubelet_configure_tls_min_version/rule.yml

```diff
@@ -0,0 +1,93 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+platform: ocp4
+
+title: "Ensure Kubelet is configured to use secure TLS version even for the minimum version"
```

Comment:
nits: As this rule check for allowed version ranges, would it be better to state as  "Ensure Kubelet is configured with allowed TLS versions" ?

Reply:
Yeah, but since TLS1.0 and TLS1.1 can also be configured, I think it would not convey our objective to configure with only secure TLS version, 1.2 or greater.

Reply:
"allowed TLS versions"  would mean, the valid versions 1.2 and above. TLS1.0 and TLS1.1 though valid but are not allowed.

Reply:
I agree with Trilok fwiw, the wording could be improved a little

Reply:
comment incorporated